tsl/test/isolation/specs/remote_create_chunk.spec.in

```diff
@@ -91,15 +87,6 @@ step "s3_conditions_locks" {
 }
 
 session "s4"
-step "s4_wait_all"  {
-	SELECT * FROM debug_waitpoint_id('find_or_create_chunk_start');
-	SELECT * FROM debug_waitpoint_id('find_or_create_chunk_created');
-	SELECT * FROM debug_waitpoint_id('find_or_create_chunk_found');
-	SELECT debug_waitpoint_enable('find_or_create_chunk_start');
-	SELECT debug_waitpoint_enable('find_or_create_chunk_created');
-	SELECT debug_waitpoint_enable('find_or_create_chunk_found');
-}
-
 step "s4_wait_start"           { SELECT debug_waitpoint_enable('find_or_create_chunk_start'); }
```

Comment:
Why are these tests removed for all versions?

Reply:
this is not removed but was dead code, PG13 warns about those leading to test failures/different test output